samples/word/01-basics/basic-doc-assembly.yaml

```diff
@@ -100,15 +94,13 @@ script:
                         }
                         return context.sync();
                     })
-                    .catch(function (error) {
-                        console.log(error);
-                    })
-                });
+                    .catch(OfficeHelpers.Utilities.log);
+            });
         }
 
         function addFooter() {
             Word.run(function (context) {
-            // Create a proxy collection for the sections collection.
+                // Create a proxy collection for the sections collection.
```

Comment:
At least on the Excel side, we've removed most of the "creates a proxy object..." comments